ibazel/ibazel_test.go

```diff
@@ -283,6 +290,41 @@ func TestIBazelRun_notifyPreexistiingJobWhenStarting(t *testing.T) {
 	}
 }
 
+func TestIBazelRun_livereload(t *testing.T) {
+	i, err := New()
+	if err != nil {
+		t.Errorf("Error creating IBazel: %s", err)
+	}
+	defer i.Cleanup()
+
+	i.run("//path/to:target")
+
+	livereloadUrl := os.Getenv("IBAZEL_LIVERELOAD_URL")
+	var validUrl = regexp.MustCompile(`^http\:\/\/localhost\:[0-9]+\/livereload\.js\?snipver\=1$`)
```

Comment:
I would prefer you use the more idiomatic golang

`validUrl := regexp.MustCompile("^http\:\/\/localhost\:[0-9]+\/livereload\.js\?snipver\=1$")`

2 changes in there.

1) Using `:=` to define and set a variable in one pass
2) Using double quotes for strings. backtick is great but it is easy to accidentally turn into a multi-line string.

Reply:
K. Switched to :=. Thanks.
Trying double quotes breaks the compile however with the error ` ibazel/ibazel_test.go:303:40: unknown escape sequence`. Might be that backticks are required for regexp. https://golang.org/pkg/regexp/. Not sure as I'm new to Go.

Reply:
You can double escape and it will work: `validUrl := regexp.MustCompile("^http\\:\\/\\/localhost\\:[0-9]+\\/livereload\\.js\\?snipver\\=1$")`

Reply:
That works. Thanks.